### ISSUE TITLE
[Flatpak] Does not appear on screen

### ISSUE THREAD

**Issue by mj0123456789** [sindresorhus/caprine#1698]
Hello,

Config : Linux Mint 20.2 Cinnamon
Installation as Flatpak (2.54.1)

I was impressed by first trial a couple of hours ago.
I try to use it today: nothing appears.

ls -alx | grep caprine => plenty of caprine items.
killall caprine
Try to launch the app => nothing comes.

Tried to uninstall AND reinstall (first via GUI software in Mint second via CLI)
=> Still the same problem.
Same process plus remove caprine directory still existing after uninstall var/lib/flatpak/app (and .removed)
=> Still the same problem.

Regards,

Marc

**Comment by RDKRACZ** [comment]
have you tried launching Caprine from the terminal to get a log?
Maybe it's just launching minimalized - to the tray icon?

**Comment by mj0123456789** [comment]
Hi RDKRACZ,
Thx for your answer.
I thought about that since I've changed the parameter to launch minimalized, with icon in the toolbar => when clicking on the icon, it does not appear either.
So I tried uninstall / reinstall and deleting directories since I thought it would start with defaults.
If there is a way to reinitialize to defaults param, let me know.
Or maybe an equivalent to "apt-get purge" (rather then "apt-get remove") for flatpaks.
Regards.
